src/views/Player/Player.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import { fetchPlayerById } from '../../services/players';
+import PlayerDetails from '../../components/Player/PlayerDetails';
+
+export default function Player(props) {
+  const [player, setPlayer] = useState([]);
```

Comment:
player ends up as an object, would make more sense to set the initial state to an empty object `{}`